libs/portal/pages/schedule.tsx

```
@@ -0,0 +1,148 @@
+    <Layout isCompleteProfile={false}>
+      <section className="max-w-screen-xl mx-auto p-5 sm:px-10 py-40 pt-10">
+        <GlassmorphicCard className="row-span-2 p-5 text-center mb-5">
+          <Typography variant="h2">SCHEDULE</Typography>
+        </GlassmorphicCard>
+
+        <div className="flex flex-col gap-5">
+          {/* DAY SELECTION */}
+          <div className="flex flex-row gap-3 justify-center">
+            {days.map(day => (
+              <button key={day.date} onClick={() => setSelectedDay(day.date)}>
+                <GlassmorphicCard
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)